File: specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-06-01/examples-js/ListSqlPoolOperations.js
const { SynapseManagementClient } = require("@azure/arm-synapse");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Gets a list of operations performed on the SQL pool.
 *
 * @summary Gets a list of operations performed on the SQL pool.
 * x-ms-original-file: specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-06-01/examples/ListSqlPoolOperations.json
 */
async function listTheSqlAnalyticsPoolManagementOperations() {
  const subscriptionId =
    process.env["SYNAPSE_SUBSCRIPTION_ID"] || "00000000-1111-2222-3333-444444444444";
  const resourceGroupName = process.env["SYNAPSE_RESOURCE_GROUP"] || "sqlcrudtest-7398";
  const workspaceName = "sqlcrudtest-4645";
  const sqlPoolName = "testdb";
  const credential = new DefaultAzureCredential();
  const client = new SynapseManagementClient(credential, subscriptionId);
  const resArray = new Array();
  for await (let item of client.sqlPoolOperations.list(
    resourceGroupName,
    workspaceName,
    sqlPoolName
  )) {
    resArray.push(item);
  }
  console.log(resArray);
}

File: specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-03-01/examples-java/ListSqlPoolOperations.java

/**
 * Samples for SqlPoolOperations List.
 */
public final class Main {
    /*
     * x-ms-original-file:
     * specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-03-01/examples/ListSqlPoolOperations.json
     */
    /**
     * Sample code: List the Sql Analytics pool management operations.
     * 
     * @param manager Entry point to SynapseManager.
     */
    public static void
        listTheSqlAnalyticsPoolManagementOperations(com.azure.resourcemanager.synapse.SynapseManager manager) {
        manager.sqlPoolOperations().list("sqlcrudtest-7398", "sqlcrudtest-4645", "testdb",
            com.azure.core.util.Context.NONE);
    }
}
